.travis.yml

```diff
@@ -1,13 +1,34 @@
-language: swift
-os: osx
 osx_image: xcode10
-
+language: swift
+cache:
+  directories:
+  - Carthage
+env:
+  global:
+  - FRAMEWORK_NAME=MnemonicKit
+before_install:
+- gem install slather
+- brew update
+- brew outdated carthage || brew upgrade carthage
+- carthage bootstrap --platform iOS --cache-builds
+- pod install || pod install --repo-update
+before_deploy:
+- carthage build --no-skip-current --platform iOS --cache-builds
+- carthage archive $FRAMEWORK_NAME
+- pod trunk push
```

Comment:
I don't  think we want to push to Cocoapods trunk here. Besides that Cocoapods won't have an API key available, we shouldn't publish the effects of any build.

Reply:
Likewise, you should consider if `carthage archive` is really what you want to do here (I'm less familiar)